runtime/vm/MHInterpreter.inc

```diff
@@ -545,6 +537,7 @@ VM_MHInterpreter::dispatchLoop(j9object_t methodHandle)
 			}
 			break;
 		}
+#if defined(J9VM_OPT_METHOD_HANDLE) && (JAVA_SPEC_VERSION >= 11)
```

Comment:
I believe this whole file is undefrthe opt_method_handle ifdef already.

Reply:
I checked: it is compiled for Java17+, but perhaps it should not be. A fix for that should be separate.